index.html

```diff
@@ -45,7 +45,7 @@ <h2>Data</h2>
     <div class="col-md-13">
       <h3>Give</h3>
       <p class="lead">Please partner with us to support biodiversity informatics research, taxonomy, and natural history collections. Contributions impact efforts, from local-to-global, to understand, preserve, and responsibly use the Earth's vast biodiversity. Questions?  <a href="/about">Contact us</a>.</p>
-      <button class="btn btn-primary btn-lg"> <a href="https://www.givecampus.com/campaigns/49638/donations/new">Donate</a> </button>
```

Comment:
@jlpereira Text is blue on a blue button what are we missing here?

Reply:
Remove `button` tag and assign the classes to `a` tag